compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -1212,8 +1216,8 @@ void ConvertToLLVMPass::runOnOperation() {
   arith::populateArithToLLVMConversionPatterns(converter, patterns);
   populateVectorToSCFConversionPatterns(patterns);
   populateVectorToLLVMMatrixConversionPatterns(converter, patterns);
-  populateVectorToLLVMConversionPatterns(converter, patterns,
-                                         reassociateFpReductions);
+  populateVectorToLLVMConversionPatterns(
+      converter, patterns, targetReassociateFpReductions.getValue());
```

Comment:
This is the global flag, right? Should we rename it to something like `ireeTargetReassociate...` so that it doesn't match the new local flag?

Reply:
I am following the convention used in the other options - prefix it with "target"